packages/web-pkg/src/components/SideBar/FileSideBar.vue

```diff
@@ -334,6 +345,39 @@ export default defineComponent({
       sharesStore.setLoading(false)
     }).restartable()
 
+    watch(
+      () => [...unref(panelContext).items, props.isOpen],
+      async () => {
```

Comment:
Hmm now we have 2 watchers on the very same properties. Did you check that it doesn't have any negative effects, especially performance-wise? Not sure if we should do this. Maybe @dschmidt knows?

Reply:
Well obviously it's better to have only one watcher, but if it's not in a component that is used hundreds of times in a web page I doubt it has a real performance impact. What I'm usually more worried about (havent checked this case tho) is that it becomes hard to reason about as it might not be clear which handler is executed first